app/common/enterprise-util.ts

```diff
@@ -1,8 +1,10 @@
+import {app} from "electron/main";
```

Comment:
This does not work. We’re in `app/common` here, which is used from both the main process and the renderer process. We’re not allowed to use main process APIs. In practice, we get `app === undefined` in the renderer process.

Reply:
Would #1430 the approach for rectifying this?